database/insert_test_data.sql

```diff
@@ -0,0 +1,48 @@
+-- INSERT TEST DATA
+-- Inserting data into the "users" table
+INSERT INTO onboarding_schema.users (id, name) VALUES
+(5, 'Nicolas Wildbaum'),
+(6, 'Deborah Laufer'),
+(7, 'Micaela Wildbaum');
+
+
+-- Inserting data into the "items" table
+INSERT INTO onboarding_schema.items (id, name) VALUES
+(101, 'Item A'),
+(102, 'Item B'),
+(103, 'Item C');
+
+-- Inserting data into the "purchases" table
+INSERT INTO onboarding_schema.purchases ("user", item) VALUES
+(4, 102),
```

Comment:
you are using user ids like 4,3,2,1 that where not created above in the user INSERT